magpie/ui/utils.py

```diff
@@ -567,7 +567,7 @@ def create_user(self, data):
         if len(user_name) > get_constant("MAGPIE_USER_NAME_MAX_LENGTH", self.request):
             data["invalid_user_name"] = True
             data["reason_user_name"] = "Too Long"
-        if user_name in [usr["user_name"] for usr in user_details]:
+        if user_name.lower() in [usr["user_name"].lower() for usr in user_details]:
```

Comment:
Not needed here. Enforce it directly from the DB/API.

Reply:
What is the point of these checks then?

Reply:
The best would be to avoid duplicating the logic in the UI and leave all validations on the API side that can report more specific error messages. 
However, given how small this change is, it is fine to leave it as is.
Disregard my previous comment.